app/javascript/components/form/components/form-section-field.jsx

```diff
@@ -47,7 +47,9 @@ const FormSectionField = ({ checkErrors, field, formMethods, formMode, disableUn
   return (
     handleVisibility() || (
       <ConditionalWrapper condition={Boolean(WrapWithComponent)} wrapper={WrapWithComponent}>
-        <div className={classes}>{renderField}</div>
+        <div className={classes} data-testid="form-section-field">
```

Comment:
The `FormSectionField` is a component used by all fields. I think this `data-testid` should be placed in the `FieldList` to make sure is rendered when the field is a subform.